src/components/views/messages/MImageBody.tsx

```diff
@@ -496,8 +497,14 @@ export default class MImageBody extends React.Component<IBodyProps, IState> {
     // Overidden by MStickerBody
     protected getFileBody(): string | JSX.Element {
         if (this.props.forExport) return null;
-        // We only ever need the download bar if we're appearing outside of the timeline
-        if (this.props.tileShape) {
+        /**
```

Comment:
block comments != jsdoc 😇 
```suggestion
        /*
```